### PR TITLE
Modified job status icon and job page style

### PR DESCRIPTION
**Pull request by fenghuajia**

### CODE REVIEW

**Comment by fenghuajia** [comment]
image
image

File: src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx [hunk 1]
```diff
@@ -169,22 +168,25 @@ export default function Table() {
...
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconStyle = {
+        color: 'white', borderRadius: '50%', 
         backgroundColor: {
-          Waiting: '#FCD116',
-          Running: '#0071BC',
-          Stopping: '#0071BC',
-          Succeeded: '#7FBA00',
-          Failed: '#E81123',
+          Waiting: '#F9B61A',
```
Comment (sunqinzheng) [inline src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx]: status badge component has been extracted to src/app/components/status-badge.jsx.
Please require and tweak the common component.

File: src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx [hunk 2]
```diff
@@ -169,22 +168,25 @@ export default function Table() {
         Failed: MessageBarType.remove,
         Stopped: MessageBarType.blocked,
       }[statusText];
-      const rootStyle = {
+      
+      const rootStyle = {backgroundColor: 'transparent'};
+      /** @type {import('@uifabric/styling').IStyle} */
+      const iconContainerStyle = {marginTop: 8, marginBottom: 8, marginLeft: 8};
```
Comment (sunqinzheng) [inline src/webportal/src/app/job/job-view/fabric/JobList/Table.jsx]: Please use fabric's spacing constants instead.
example: https://github.com/Microsoft/pai/blob/bd08823c0be7fea9fb55efe42e5a48e06724ea29/src/webportal/src/app/home/home/job-status.jsx#L51
current value: https://github.com/Microsoft/pai/blob/bd08823c0be7fea9fb55efe42e5a48e06724ea29/src/webportal/src/app/components/theme.js#L22

File: src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx [hunk 3]
```diff
@@ -336,7 +338,7 @@ export default class TaskRoleContainerList extends React.Component {
     const {className, style, taskInfo} = this.props;
     const status = isNil(taskInfo) ? this.generateDummyTasks() : taskInfo.taskStatuses;
     return (
-      <div className={className} style={{backgroundColor: theme.palette.white, ...style}}>
+      <div className={className} style={{backgroundColor: 'theme.palette.white', ...style}}>
```
Comment (sunqinzheng) [inline src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx]: `theme.palette.white` is a javascript string variable. It doesn't need quotation marks

File: src/webportal/src/app/job/job-view/fabric/job-detail/components/summary.jsx [hunk 4]
```diff
@@ -208,16 +209,41 @@ export default class Summary extends React.Component {
...
+      color: 'white', borderRadius: '50%', 
+      backgroundColor: {
+        Waiting: '#F9B61A',
+        Running: '#579AE6',
+        Stopping: '#579AE6',
+        Succeeded: '#54D373',
+        Failed: '#E06260',
+        Stopped: '#B1B5B8',
+      }[getHumanizedJobStateString(jobInfo)],
+        transform: getHumanizedJobStateString(jobInfo) == 'Failed' ? 'rotate(90deg)' : 'rotate(0deg)',
```
Comment (sunqinzheng) [inline src/webportal/src/app/job/job-view/fabric/job-detail/components/summary.jsx]: Why not use a rotated failed icon image?
And I prefer using a svg icon

**Pull request closed without merging**